### PR TITLE
Moyo Header develop

### PR DESCRIPTION
**Pull request by Sergei-Nikolaienko**
https://Sergei-Nikolaienko.github.io/layout_moyo-header/
https://Sergei-Nikolaienko.github.io/layout_moyo-header/report/html_report/

### CODE REVIEW

**Review by AlisaVlasova — CHANGES_REQUESTED** [review]
image

**Comment by Sergei-Nikolaienko** [comment]
Fixed as much as I can, 2 tests won't pass for some reason, however.

File: src/style.css [hunk 1]
```diff
@@ -1 +1,63 @@
...
+  display: flex;
+  height: 60px;
+  align-items: center;
+  justify-content: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: black;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0 0 0 20px;
```
Comment (SanyaBratashchuk) [inline src/style.css:43]: remove margin for first element

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Tests fail because of left padding on ul and left padding on the first link. Nav shouldn't contain this empty space 
image

**Review by qoa11a — APPROVED** [review]

**Pull request closed without merging**